### PR TITLE
removes redundant usage of FRONTEND_HOST variable

### PR DESCRIPTION
**Pull request by mgogoulos**
* allows a MediaCMS portal to load on all ips (not only the one specified as FRONTEND_HOST)
* removes the dependency of FRONTEND_HOST - it's now there mainly for ssl certificate installation (on setup using install.sh) and in order to appear as the portal url on emails

Needs testing on all cases (install.sh installation, Docker install)

### CODE REVIEW

**Comment by swiftugandan** [comment]
Will test the docker setup ...

**Review by swiftugandan — APPROVED** [review]
I have tested this on docker and seems to work really well.

File: templates/config/installation/site.html [hunk 1]
```diff
@@ -1,8 +1,8 @@
 MediaCMS.site = {
  	id: 'mediacms',
     title: "{{PORTAL_NAME}}",
-    url: '{{FRONTEND_HOST}}/',
-    api: '{{FRONTEND_HOST}}/api/v1',
+    url: '{{FRONTEND_HOST}}',
+    api: '{{FRONTEND_HOST}}api/v1',
```
[the remark this answers is not in the document]
Comment (mgogoulos) [reply templates/config/installation/site.html:5]: It's there - https://docs.djangoproject.com/en/3.1/ref/request-response/#django.http.HttpRequest.build_absolute_uri

**Comment by mgogoulos** [comment]
> I have tested this on docker and seems to work really well.

glad to read this!

**Pull request merged**